### ISSUE TITLE
adapt-contrib-mcq: NVDA navigation problem

### ISSUE THREAD

**Issue by canstudios-nicolaw** [adaptlearning/adapt_framework#2463]
When using the MCQ component in conjunction with the tutor extension. 2 attempts were set on this question. On the 1st attempt after closing the notify popup focus is on the reset button. However, it cannot be selected by the spacebar/enter. The navigation seems stuck. On the last attempt after closing the notify popup feedback navigation gets lost. Screenreader just says blank so you are not even sure of your location. Even when using tab to get focus back to the same location, buttons cannot be selected by using spacebar/enter. The only way to get back control is to refresh the browser. 

### Your environment
* 0.8.1 AT & 4.2.0 FW
* IE11 
* Windows 10
* MCQ 3.0.0

### Steps to reproduce
- Create an MCQ with two attempts
- Add the tutor extension
- Attempt to navigate through the course using NVDA on IE11

**Comment by oliverfoster** [comment]
Closed as stale. Possibly known issue.
